### PR TITLE
Minor repository improvements/fixes

### PR DESCRIPTION
**Pull request by Panquesito7**
Things added/changed:

- Minor repository improvements/fixes.

### CODE REVIEW

File: CONTIBUTING.md [hunk 1]
```diff
@@ -10,70 +10,69 @@ First off, thank you for taking your time to contribute to this project! 🎉
...
-**11.** Push the committed changes in your feature branch to your remote repo.
+**11.** Push the committed changes in your feature branch to your remote repository.
 
-```
+```bash
 git push -u origin <your_branch_name>
 ```
 
-**12.** To create a pull request, click on `compare and pull requests`.
+**12.** To create a pull request, click on the big breen `New pull request` button.
```
Comment (sidwebworks) [inline CONTIBUTING.md:67]: @Panquesito7 little typos

**Review by sidwebworks — APPROVED** [review]
Just one typo, where I commented rest looks good to me.

**Comment by Panquesito7** [comment]
Fixed. 🙂

**Comment by Panquesito7** [comment]
Will be merging now. If there's anything that needs to be fixed, we can make another PR later. 🙂

**Comment by sidwebworks** [comment]
Sorry forgot to view my notifications, Haha it was all good so I thought you'll just merge it

**Pull request merged**